<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konami Socket Program</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ServerTab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QGroupBox" name="DesiredConnectionBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Desired Connection</string>
      </property>
      <widget class="QRadioButton" name="DefaultRadioButton_Server">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="CustomRadioButton_Server">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="IPAddressBox_Server">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter IP Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PortBox_Server">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Port</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="StartServerButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="ScrollArea_Server">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>511</width>
        <height>351</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>509</width>
         <height>349</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="TextArea_Server">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>511</width>
          <height>351</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextEditorInteraction</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="ClientTab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QGroupBox" name="DesiredConnection_Client">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Desired Connection</string>
      </property>
      <widget class="QRadioButton" name="DefaultRadioButton_Client">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="CustomRadioButton_Client">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="IPAddressBox_Client">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter IP Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PortBox_Client">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Port</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="StartClientButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="ScrollArea_Client">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>511</width>
        <height>351</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>509</width>
         <height>349</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="TextArea_Client">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>511</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="SendXMLArea">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>169</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Send XML File</string>
      </property>
      <widget class="QPushButton" name="BrowseXMLButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse XML</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xmlFileNameBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Unit Test</string>
     </attribute>
     <widget class="QPushButton" name="UnitTestStartButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Tests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UnitTestStopButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Tests</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>701</width>
        <height>311</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>699</width>
         <height>309</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="UnitTestTextArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QProgressBar" name="UnitTestProgressBar">
      <property name="geometry">
       <rect>
        <x>427</x>
        <y>10</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKonami_Socket_Program">
    <property name="title">
     <string>Konami Socket Program</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuv1">
    <property name="title">
     <string>v1</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tralyn_Le">
    <property name="title">
     <string>- Tralyn Le</string>
    </property>
   </widget>
   <addaction name="menuKonami_Socket_Program"/>
   <addaction name="menuv1"/>
   <addaction name="menu_Tralyn_Le"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
